src/sass/_mixins.scss

```diff
@@ -82,6 +82,18 @@
   }
 }
 
+@mixin height($size_in_px) {
+  height: #{$size_in_px}px;
+  height: #{typography.get-rem-from-px($size_in_px)}rem;
+}
+
+@mixin constrain-height($size_in_px, $extreme) {
```

Comment:
I can't find a use of this, was it copied across from journal by mistake or did you bring it just for symmetry?

Reply:
I did bring it over and started to use it but decided it was only being used once so will remove it